pkg/util/util_test.go

```diff
@@ -353,6 +353,35 @@ func TestGetPortSpecified(t *testing.T) {
 	assert.Equal(t, int32(6831), GetPort("--processor.jaeger-compact.server-host-port=", args, 1234))
 }
 
+func TestGetAdminPortDefault(t *testing.T) {
```

Comment:
This looks good, but you could have used a table test here, with the args, the default port and the expected outcome as the test's struct:

https://github.com/golang/go/wiki/TableDrivenTests

Reply:
That looks better, will change to table test now